bayesian/utility/db_gateway.py

```diff
@@ -217,12 +217,13 @@ class RdbAnalyses:
     Provides interfaces to save and read request post data for stack analyses v2.
     """
 
-    def __init__(self, request_id, submit_time=None, manifest=None, deps=None):
+    def __init__(self, request_id, submit_time=None, manifest=None, deps=None, user_id=None):
```

Comment:
```suggestion
    def __init__(self, request_id):
```

Change the constructor to only take in parameters that are used by all functions.  The other parameters are specific to the save_post_request method and can be passed as parameters to that.

Reply:
Okay .Makes sense.

Reply:
Resolved.